analyses/cell-type-analysis/scripts/01_read-data.R

```diff
@@ -0,0 +1,14 @@
+library(SingleCellExperiment)
+
+sce <- readRDS("../../data/current/SCPCL999990_processed.rds")
```

Comment:
I would recommend using `file.path()` here to define the path before reading in the file.

Reply:
Thanks for this reminder! I will add a separate section in the script that uses `file.path()` to define input and output files.